lib/daru/io/exporters/linkages/csv.rb

```diff
@@ -3,16 +3,42 @@
 module Daru
   class DataFrame
     class << self
-      def to_csv(opt={})
-        exporters = Daru::IO::Exporters
-        case opt[:gem]
-        when :fastest_csv
-          exporters::FastestCSV.load opt[:str]
-        when :rcsv
-          exporters::RCSV.load opt[:str]
-        else
-          exporters::CSV.load opt[:str]
-        end
+      # Write this DataFrame to a CSV file.
+      #
+      # == Arguements
+      #
+      # * filename - Path of CSV file where the DataFrame is to be saved.
+      #
+      # == Options
+      #
+      # * convert_comma - If set to *true*, will convert any commas in any
+      # of the data to full stops ('.').
+      # All the options accepted by CSV.read() can also be passed into this
+      # function.
```

Comment:
As Lokesh have said, let's use proper YARD+Markdown docs from the very beginning.